frontend/packages/ceph-storage-plugin/src/components/ocs-install/existing-cluster-modal.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Trans, useTranslation } from 'react-i18next';
+import { match as RouteMatch } from 'react-router';
+import { Button } from '@patternfly/react-core';
+import { Modal } from '@console/shared/src/components/modal';
+import { ClusterServiceVersionModel } from '@console/operator-lifecycle-manager/src';
+import { history, resourcePathFromModel } from '@console/internal/components/utils';
+import { ListKind, referenceForModel } from '@console/internal/module/k8s';
+import { StorageClusterKind } from '../../types';
+import { OCSServiceModel } from '../../models';
+
+export const ExistingClusterModal: React.FC<ExistingClusterModalProps> = ({
```

Comment:
Can we make this a default export of this file?

Reply:
I used named because I want the name to be consistent all over.
Do you think we should make it default considering its  a single component ?

Reply:
yes

Reply:
I have updated but as a general rule I prefer to do named exports irrespective of a single file.
It messes with typos and intellisense, automcomplete. If this component want to be reused then anyone can export like `ExistingClusterModal  from './file'` or `existingClusterModal from './file'`